dashboard/app/controllers/programming_expressions_controller.rb

```diff
@@ -137,8 +137,7 @@ def clone
   def docs_show
     if DCDO.get('use-studio-code-docs', false)
       return render :not_found unless @programming_expression
-      @programming_environment_categories = @programming_expression.programming_environment.categories_for_navigation
-      return render :show
+      return redirect_to(programming_environment_programming_expression_path(@programming_expression.programming_environment.name, @programming_expression.key))
```

Comment:
I forget, do we need a `return` before `redirect_to`?  (I see we have a `return` here but not up in `programming_environments_controller.rb`.)

Reply:
I apparently used slightly different patterns 🙃 programming_environments_controller uses an if/else so it doesn't need a return, whereas this controller action is not. I'll pick one and standardize the two